extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -118,71 +119,106 @@ public function loadData() {
 
 			/* initial conditions for SQL query */
 			$where = [
-					'wiki_id' => $this->mCityId,
-					"user_id > 0",
+					self::TABLE . '.wiki_id' => $this->mCityId,
+					self::TABLE . ".user_id > 0",
 					'user_is_closed' => 0
 			];
 
-			/* filter: groups */
-			if ( !empty( $this->mFilterGroup ) && is_array( $this->mFilterGroup ) ) {
-				$whereGroup = array();
-				foreach ( $this->mFilterGroup as $group ) {
-					if ( !empty($group) ) {
-						if ( $group == Listusers::DEF_GROUP_NAME ) {
-							$whereGroup[] = " all_groups = '' ";
-						} else {
-							$whereGroup[] = " all_groups " . $dbs->buildLike( $dbs->anyString(), $group );
-							$whereGroup[] = " all_groups " . $dbs->buildLike( $dbs->anyString(), sprintf("%s;", $group), $dbs->anyString() );
-						}
-					}
-				}
-
-				if ( !empty( $whereGroup ) ) {
-					$where[] = implode( " or ", $whereGroup );
-				}
-			}
-
 			/* filter: user ID  */
 			if ( !empty( $this->mUserId ) ) {
-				$where[] = " user_id = ". intval( $this->mUserId );
+				$where[] = self::TABLE . ".user_id = $this->mUserId";
 			}
 
 			/* filter: number of edits */
 			if ( !empty( $this->mEditsThreshold ) ) {
-				$where[] = " edits >= ". intval( $this->mEditsThreshold );
+				$where[] = "edits >= $this->mEditsThreshold";
 			}
 
-			/* number of records */
-			$data['cnt'] = (int) $dbs->selectField(
-				self::TABLE,
-				'count(*)',
+			/* filter: groups */
+			if ( !empty( $this->mFilterGroup ) && is_array( $this->mFilterGroup ) ) {
```

Comment:
Perhaps this would be a bit simpler? There's a check if filtered groups are empty, right?
```suggestion
			if ( is_array( $this->mFilterGroup ) ) {
```